x/distribution/spec/04_messages.md

```diff
@@ -74,19 +52,16 @@ func WithdrawDelegationReward(delegatorAddr, validatorAddr, withdrawAddr sdk.Acc
 ```
 
 
-## MsgWithdrawValidatorRewardsAll
+### Withdraw Validator Rewards All
```

Comment:
should we rephrase this since there's no more actual message for that? Like "Withdraw multiple rewards", or maybe just get rid of this sub-section?

Reply:
unresolving